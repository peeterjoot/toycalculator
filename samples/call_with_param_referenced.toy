FUNCTION bar ( INT16 w )
{
    PRINT w;
    RETURN;
};

PRINT "In main";
CALL bar( 3 );
PRINT "Back in main";
