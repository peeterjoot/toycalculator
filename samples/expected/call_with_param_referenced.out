In main
3
Back in main
